package.json

```diff
@@ -68,6 +68,8 @@
     "webpack-dev-server": "^3.1.14"
   },
   "dependencies": {
+    "node-forge": "^0.10.0",
```

Comment:
Why do we add them to the package.json?

Isnt it enough to bump them in the pkg.lock?